.travis.yml

```diff
@@ -31,6 +35,8 @@ deploy:
     tags: true
     repo: mozilla/addons-linter
     branch: master
+    node_js: 'v10.21.0'
```

Comment:
@willdurand did you already checked if there will be a visible failure if at some point we do remove nodejs v10.21.0 from the job matrix (the one at line 3) and we forget to update the node_js version referenced here?

Reply:
No, not sure how to test this.

I don't think we'd forget that but we could mitigate the risk by adding a comment at the top of the file?

Reply:
> No, not sure how to test this.

I guess that one easy way to test it is to refer a node_js version that is supported by travis but not listed in the job matrix at line 3.
 
> I don't think we'd forget that but we could mitigate the risk by adding a comment at the top of the file?

Sure, I'll be ok with just adding a comment too

Reply:
> I guess that one easy way to test it is to refer a node_js version that is supported by travis but not listed in the job matrix at line 3.

@rpl looks like travis does not care..

Reply:
I added a comment.

Reply:
Note: I also tried to use the `travis lint` command but it did not raise any error either. It could raise an error at runtime, but we never know with travis..